inspirehep/modules/workflows/workflows/author.py

```diff
@@ -53,7 +53,7 @@
 
 
 SEND_TO_LEGACY = [
-    send_robotupload(mode="insert", priority_config_key='LEGACY_ROBOTUPLOAD_PRIORITY_AUTHOR'),
```

Comment:
why not simply hardcode `replace` here instead?

Reply:
No. It's ugly and it's simplifying nothing.

Reply:
It prevents bugs. There's no need to make it metadata dependent, as it can be inferred by where it's called in the workflow: we know that `replace` should work because it already has a control number. If there is no control number at that stage, it will be rejected and we'll notice there's a bug.

Reply:
:/ ok... Ugly but ok.